home/dot_local/bin/mackup/executable_mackup.tmpl

```diff
@@ -0,0 +1,33 @@
+{{- if ne .host.distro.family "windows" -}}
+#!/usr/bin/env bash
+# @file ~/.local/bin/mackup
+# @brief Creates a wrapper around `mackup` to add support for XDG directories
+# @description
+#   This script is a wrapper around the `mackup`. `Mackup` does not support XDG specification
+#   for its configuration. The script symlinks the `.config/mackup/.mackup/` folder to `~/.mackup` and
+#   the `.config/mackup/.mackup.cfg` file to `~/.mackup.cfg` before executing `mackup`. The symlinks
+#   are deleted after mackup runs.
+#
+#   [mackup](https://github.com/lra/mackup) is a tool that can help in keeping your application settings
+#   in sync across multiple computers. It supports many different storage solutions like Dropbox, Google Drive, etc.
+#   to store the settings and sync them across machines. It can also help in restoring settings on a new installation.
+#   A number of applications are supported out of the box. It is trivial to add more applications. Refer to the
+#   [documentation](https://github.com/lra/mackup/blob/master/doc) for details.
+
+{{ includeTemplate "universal/profile" }}
+{{ includeTemplate "universal/logg" }}
+
+ln -s "$HOME/.config/mackup/.mackup/" ~/.mackup
+ln -s "$HOME/.config/mackup/.mackup.cfg" ~/.mackup.cfg
+if command -v brew > /dev/null && brew list | grep mackup > /dev/null; then
```

Comment:
I don't think this is necessary. You should just be able to use `command -v mackup` and then use `mackup` directly.

Reply:
The below installation methods are possible and these result in mackup installed to a various locations.These additional checks are to cover the below possible installation methods and set the path accordingly. Without specifying the full path, the export `PATH="$HOME/.local/bin/mackup:$PATH"` will result in a loop.
* Homebrew based installation in Linux (installed to `/home/linuxbrew/.linuxbrew/bin`)
* Homebrew based installation in Mac
* Pip based installation where pip was installed using homebrew. Location of mackup depends on the whether this is Linux/MacOS
* Pip based installation where pip was installed using OS package manager
* Availability of pip/pip3 (Archlinux)
* Installation using pipx